src/organisms/menu/component.tsx

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+
+import { getClassName } from '@kibalabs/core';
+import { IMultiChildProps, OptionalProppedElement } from '@kibalabs/core-react';
+import styled from 'styled-components';
+
+import { IListItemTheme, ListItem } from '../../atoms';
```

Comment:
why is this an organism and not a molecule? Also the menuitem seems like it should really be a particle. this should really looke just like list and listitem i think?

Reply:
hm in general i think the challenge here is its very hard for me to tell if this is on the right track or not. i'd suggest not continuing on code and going back to the google doc - pls try document your thinking in terms of theming and particle/molecule/organism there.

Reply:
As always, should not be made in the PR, they should be made before but here there are loads of "surprises" i wasnt expecting.

Reply:
I will update the doc file with whatever I thought about the menu.